docs/_docs/tools/control-script.adoc

```diff
@@ -646,4 +646,136 @@ control.bat --change-tag _<new-tag>_
 ----
 --
 
+== Metric command
 
+Metrics command prints value(s) of the metric or metric registry from the parameter.
```

Comment:
The metrics command prints out the value of a metric or metric registry provided in the parameters list. Use the `--node-id` parameter, If you need to get a metric from a specific node. Ignite selects a random node, if the `--node-id` is not set.

Reply:
Fixed. Thanks!